aws/v18.2.0/release.diff

```diff
@@ -0,0 +1,78 @@
+# Generated with:                                                # Generated with:
+# devctl release create --name v18.1.0 --provider aws --base v | # devctl release create --name v18.2.0 --provider aws --base v
+apiVersion: release.giantswarm.io/v1alpha1                       apiVersion: release.giantswarm.io/v1alpha1
+kind: Release                                                    kind: Release
+metadata:                                                        metadata:
+  annotations:                                                     annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/c       giantswarm.io/docs: https://docs.giantswarm.io/reference/c
+  creationTimestamp: null                                          creationTimestamp: null
+  name: v18.1.0                                                |   name: v18.2.0
```

Comment:
why 18.1.0 not 18.1.1?

Reply:
My bad, the only thing what changed in v18.1.1 was aws-operator. But I fixed it 👍🏻